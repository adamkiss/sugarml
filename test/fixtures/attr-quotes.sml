.test(loop="item of [item1(), 'item2']")
