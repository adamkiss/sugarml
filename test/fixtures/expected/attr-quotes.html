<div class="test" loop="item of [item1(), 'item2']"></div>
